Marlin/temperature.cpp

```diff
@@ -175,11 +205,20 @@ static volatile bool temp_meas_ready = false;
 #endif //PIDTEMP
 
 // Init min and max temp with extreme values to prevent false errors during startup
-static int minttemp_raw[EXTRUDERS] = ARRAY_BY_EXTRUDERS( HEATER_0_RAW_LO_TEMP , HEATER_1_RAW_LO_TEMP , HEATER_2_RAW_LO_TEMP, HEATER_3_RAW_LO_TEMP);
-static int maxttemp_raw[EXTRUDERS] = ARRAY_BY_EXTRUDERS( HEATER_0_RAW_HI_TEMP , HEATER_1_RAW_HI_TEMP , HEATER_2_RAW_HI_TEMP, HEATER_3_RAW_HI_TEMP);
-static int minttemp[EXTRUDERS] = ARRAY_BY_EXTRUDERS( 0, 0, 0, 0 );
-static int maxttemp[EXTRUDERS] = ARRAY_BY_EXTRUDERS( 16383, 16383, 16383, 16383 );
-//static int bed_minttemp_raw = HEATER_BED_RAW_LO_TEMP; /* No bed mintemp error implemented?!? */
+#ifndef SINGLENOZZLE
```

Comment:
You can do a similar trick here…

    #ifdef SINGLENOZZLE
      #define HEATER_RAW(E,LOHI) HEATER_ ## E ## _RAW_ ## LOHI ## _TEMP
    #else
      #define HEATER_RAW(E,LOHI) HEATER_0_RAW_ ## LOHI ## _TEMP
    #endif
    static int minttemp_raw[EXTRUDERS] = ARRAY_BY_EXTRUDERS(HEATER_RAW(0,LO) , HEATER_RAW(1,LO) , HEATER_RAW(2,LO), HEATER_RAW(3,LO));
    static int maxttemp_raw[EXTRUDERS] = ARRAY_BY_EXTRUDERS(HEATER_RAW(0,HI) , HEATER_RAW(1,HI) , HEATER_RAW(2,HI), HEATER_RAW(3,HI));
    static int minttemp[EXTRUDERS] = ARRAY_BY_EXTRUDERS( 0, 0, 0, 0 );
    static int maxttemp[EXTRUDERS] = ARRAY_BY_EXTRUDERS( 16383, 16383, 16383, 16383 );
    //static int bed_minttemp_raw = HEATER_BED_RAW_LO_TEMP; /* No bed mintemp error implemented?!? */